moto/ec2/responses/elastic_network_interfaces.py

```diff
@@ -39,16 +40,43 @@ def delete_network_interface(self):
             return template.render()
 
     def describe_network_interface_attribute(self):
-        raise NotImplementedError(
-            "ElasticNetworkInterfaces(AmazonVPC).describe_network_interface_attribute is not yet implemented"
-        )
+        eni_id = self._get_param("NetworkInterfaceId")
+        attribute = self._get_param("Attribute")
+        if self.is_not_dryrun("DescribeNetworkInterfaceAttribute"):
+            eni = self.ec2_backend.get_all_network_interfaces([eni_id])[0]
+
+            if attribute == "description":
+                template = self.response_template(
+                    DESCRIBE_NETWORK_INTERFACE_ATTRIBUTE_RESPONSE_DESCRIPTION
+                )
+                return template.render(eni=eni)
+            elif attribute == "groupSet":
+                template = self.response_template(
+                    DESCRIBE_NETWORK_INTERFACE_ATTRIBUTE_RESPONSE_GROUPSET
+                )
+                return template.render(eni=eni)
+            elif attribute == "sourceDestCheck":
+                template = self.response_template(
+                    DESCRIBE_NETWORK_INTERFACE_ATTRIBUTE_RESPONSE_SOURCEDESTCHECK
+                )
+                return template.render(eni=eni)
+            elif attribute == "attachment":
+                template = self.response_template(
+                    DESCRIBE_NETWORK_INTERFACE_ATTRIBUTE_RESPONSE_ATTACHMENT
+                )
+                return template.render(eni=eni)
+            else:
+                raise InvalidParameterValueException(
```

Comment:
This case should be tested as well, to verify that the exception is returned in the correct format